README.md

```diff
@@ -398,3 +398,60 @@ change the UUID.
 
 You can't. This is because historical bookings still refer to that prison.
 Disable it instead (see above).
+
+## Smoke Tests
+
+Runs the happy path to book, confirm and cancel a visit.  If the app
+receives a an email in the format of
+`SMOKE_TEST_EMAIL_LOCAL_PART`@`SMOKE_TEST_EMAIL_DOMAIN`, it will not
+send that email to the prison, but will direct all mail the
+`SMOKE_TEST_EMAIL_HOST`.
+
+To start the smoke tests, set the environment variables and run
+`ruby ./smoke_test/smoke_test.rb`.
```

Comment:
I don't think that would get us anything other than complexity.  It is almost a standalone app and is run from a four or five line shell script on Jenkins.